torch/nn/modules/loss.py

```diff
@@ -802,6 +801,63 @@ def forward(self, input, target):
         return F.smooth_l1_loss(input, target, reduction=self.reduction)
 
 
+class HuberLoss(_Loss):
```

Comment:
Again, this looks like the same as `SmoothL1Loss` but for the `delta` parameter. Let's just add `delta` with default value `1.`

Reply:
I agree that programmatically, it would make more sense to simply include the delta value as an additional parameter in both the functional and module versions of Smooth_L1. I have made a new commit based on the comments. Many thanks!